horaedb/metric_engine/src/storage.rs

```diff
@@ -184,6 +191,19 @@ impl CloudObjectStorage {
             execute_stream(physical_plan, ctx.task_ctx()).context("execute sort physical plan")?;
         Ok(res)
     }
+
+    fn build_write_properties(&self) -> WriterProperties {
+        let mut builder = WriterProperties::builder()
+            .set_max_row_group_size(self.write_options.num_rows_per_row_group)
+            .set_compression(self.write_options.compression);
+
+        for (col_name, col_opt) in &self.write_options.column_options {
+            let col_path = ColumnPath::new(vec![col_name.to_string()]);
+            builder = builder.set_column_dictionary_enabled(col_path, col_opt.enable_dict);
```

Comment:
We should make sure enable_dict is default to false, so this option could be named `disable_dict`.